python_backend/core/config.py

```
@@ -32,6 +32,9 @@ class Settings(BaseSettings):
     CORS_METHODS: list = ["*"]
     CORS_HEADERS: list = ["*"]
     
+    # 피드백 보안키 설정
+    SECRET_KEY: str = "default-secret-key-for-dev"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)